app/assets/stylesheets/basic.css

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
If I recall, this is the CSS for the bulk upload zone? If so, could we add a comment explaining why it needs to be there and more about where it came from?

Reply:
I have not yet worked on this css yet, comments would be appreciated so I can understand what I'll be working with.